File: ABC/206/b_1.py
def main():
    # input
    N = int(input())

    # compute
    cnt = 1
    while True:
        if (cnt+1)*cnt//2 >= N:
            break
        else:
            cnt += 1

    # output
    print(cnt)


if __name__ == '__main__':
    main()

File: abc/089/b_1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  // input
  int N;
  string S, ans = "Three";
  cin >> N;

  // compute
  for (int i = 0; i < N; i++) {
    cin >> S;
    if (S == "Y") {
      ans = "Four";
    }
  }

  // output
  cout << ans << endl;
}
